DeviceAdapters/Zaber/Stage.cpp

```diff
@@ -294,9 +294,7 @@ int Stage::OnPort (MM::PropertyBase* pProp, MM::ActionType eAct)
 	{
 		if (initialized_)
 		{
-			// revert
-			pProp->Set(port_.c_str());
-			return ERR_PORT_CHANGE_FORBIDDEN;
+			resetConnection();

```

Comment:
Does this mean the port can be changed while the device is open?

Reply:
Yeah.